src/pages/templates/blog/blog-article-list/articleList.tsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import {
+  Box,
+  Heading,
+  Link,
+  Image,
+  Text,
+  Divider,
+  HStack,
+  Tag,
+  Wrap,
+  WrapItem,
+  SpaceProps,
+  useColorModeValue,
+  Container,
+} from '@chakra-ui/react';
+
+interface IBlogTags {
+  tags: Array<string>;
+  marginTop?: SpaceProps['marginTop'];
+}
+
+const BlogTags: React.FC<IBlogTags> = (props) => {
+  return (
+    <HStack spacing={2} marginTop={props.marginTop}>
+      {props.tags.map((tag) => {
+        return (
+          <Tag size={'md'} variant="solid" colorScheme="orange" key={tag}>
+            {tag}
+          </Tag>
+        );
+      })}
+    </HStack>
+  );
+};
+
+interface BlogAuthorProps {
+  date: Date;
+  name: string;
+}
+
+export const BlogAuthor: React.FC<BlogAuthorProps> = (props) => {
+  return (
+    <Box marginTop="2" display="flex" alignItems="center">
```

Comment:
You can probably use the `Stack` component here and define `space={2}` to avoid to add `marginLeft="2"` to all childs.